package.json

```diff
@@ -25,10 +26,13 @@
   },
   "homepage": "https://github.com/melink14/rikaikun#readme",
   "devDependencies": {
-    "eslint": "^7.5.0",
+    "byline": "^5.0.0",
+    "combined-stream2": "^1.1.2",
```

Comment:
reminder from you to me for yourself: remove this line because it's not being used

Reply:
ok!